src/libraries/System.Security.Cryptography.Pkcs/src/System.Security.Cryptography.Pkcs.csproj

```diff
@@ -14,7 +14,6 @@
     <OmitResources Condition="$(TargetFramework.StartsWith('net4'))">true</OmitResources>
     <!-- Currently the netstandard build is locked to the net472 API -->
     <AssemblyVersion Condition="$(TargetFramework.StartsWith('netstandard'))">4.0.4.0</AssemblyVersion>
-    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetsBrowser)' == 'true'">SR.SystemSecurityCryptographyPkcs_PlatformNotSupported</GeneratePlatformNotSupportedAssemblyMessage>
```

Comment:
Same as above - Crypto doesn't work on Browser, so we should be generating PNSE.

If we are really removing this - then we should also remove the `SystemSecurityCryptographyPkcs_PlatformNotSupported` string from the Strings.resx.

Reply:
```'$(TargetsBrowser)' == 'true'``` will never be true as the current tfms doesnt target browser specific config

If we to add pnse for crypto here, we need to add a new tfm.

cc @marek-safar

Reply:
cc @lewing and @steveisok as well - since they are working on adding `[UnsupportedOSPlatform]` attributes for browser.

Reply:
Also note that we have `ExcludeCurrentNetCoreAppFromPackage` so if we add NetCoreAppCurrent-Browser we need to include current tfm in the package.

Reply:
I take it we would no longer be throwing PNSE out of here?

Reply:
It's my understanding we do... I don't think any crypto is going to work on browser in 5.0.

cc @bartonjs

Reply:
Yeah, we need to make sure we're continuing to throw PNSE.

Reply:
> Also note that we have ExcludeCurrentNetCoreAppFromPackage so if we add NetCoreAppCurrent-Browser we need to include current tfm in the package.

We can add netcoreappCurrent-Browser and netcoreapp3.0-Browser, that should fix our problem

Reply:
So this package already has a `netcoreapp3.0` configuration that doesn't touch any platform-specific API: https://github.com/dotnet/runtime/tree/557c96d3ce9968118981833bbd71fd416c42e499/src/libraries/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/Pal/AnyOS

So I don't see what about this assembly that is "not supported" on `browser`.  Presumably if this ended up calling any underlying API that wasn't supported that would throw, but this could hypothetically work if the underlying API works.  I don't think we should be propagating PNSE up the stack for packages as it unnecessarily complicates the upstack libraries.

Reply:
This was previously marked as PNSE, so I was carrying the assumption that something was not supported.  If that's not the case, then I agree we shouldn't be propagating up the stack.

Reply:
From https://github.com/dotnet/runtime/pull/38379

> Since the entire set of System.Security.Cryptography.* tests has been disabled on WASM recently in #37723, we need to let everyone know that those areas are not quite supported on WASM at the moment. In that case System.Security.Cryptography classes could throw PlatformNotSupportedException until some solution comes (#37669)

cc @MaximLipnin 

Are we going to enable the Pkcs tests on Browser then?

Also, this can affect what APIs are getting `[UnsupportedOSPlatform("browser")]` attributes put on them. My understanding from @lewing is that we are generating those attributes from methods/assemblies that throw PNSE.

Reply:
I wrote those issues thinking we should throw PNSE wherever possible :-).  

@eerhardt You also raise a good point.  Will the analyzer for `[UnsupportedOSPlatform("browser")]` want to flag the specific crypto api's?

Reply:
Is there something that we need to do in this pr ?

Reply:
When running System.Security.Cryptography.Pkcs.Tests library test suite on Browser Wasm, there is the following result:
`Tests run: 813, Errors: 0, Failures: 728, Skipped: 19. Time: 2.13732s`.
Most of the failures are `System.PlatformNotSupportedException : System.Security.Cryptography.Encoding is not supported on this platform.` and `System.PlatformNotSupportedException : System.Security.Cryptography.Algorithms is not supported on this platform.`

Reply:
> Is there something that we need to do in this pr ?

@Anipik I don't think so.

Reply:
Is the SystemSecurityCryptographyPkcs_PlatformNotSupported resource still being used? If not can you delete it?

@steveisok - will these APIs get ‘UnsupportedOSPlatform(“browser”) attributes on them?

Reply:
They already have - https://github.com/dotnet/runtime/pull/40612